CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+23.08 (31.07.2023)
+------------------------------------------------------------------------
+- Fix [#2056] Unintended Map Scrolling When Window Scaling is Modified 
+- Fix [#2005] Right-clicking when display scaling is set to a fractional percent causes random scrolling of view
```

Comment:
#2056 is describing the same issue as #2005. You can remove the first line, leaving the entry for #2005.